source/jormungandr/tests/ptref_tests.py

```diff
@@ -408,7 +408,7 @@ def test_line_with_active_disruption(self):
 
     def test_line_codes(self):
         """test line formating"""
-        response = self.query_region("lines/line:A?show_codes=true")
+        response = self.query_region("lines/line:A?")
```

Comment:
Maybe `?` can be removed too?

```suggestion
        response = self.query_region("lines/line:A")
```

Reply:
done in https://github.com/hove-io/navitia/pull/3770/commits/203a08d3f5aa5f1edb34e98bc33b7a62522e351a